numpy/ma/tests/test_core.py

```diff
@@ -4858,10 +4858,12 @@ def test_immutable(self):
         assert_raises(ValueError, operator.setitem, view.data, (), 1)
         assert_raises(ValueError, operator.setitem, view.mask, (), False)
 
-    @dec.knownfailureif(sys.version_info.major == 2, "See gh-9751")
     def test_coercion_int(self):
         a_i = np.zeros((), int)
         assert_raises(MaskError, operator.setitem, a_i, (), np.ma.masked)
+        def coercion_int():
+            int(np.ma.masked)
+        assert_raises(MaskError, coercion_int)
```

Comment:
Better as `assert_raises(MaskError, int, np.ma.masked)`. Here you _do_ need a test for `long` as well.

I think there might already be a test for this kind of thing elsewhere

Reply:
@eric-wieser Nope a test for this doesn't exist yet.